src/main/java/org/apache/xmlbeans/impl/store/Cursor.java

```diff
@@ -201,7 +201,7 @@ private void insertNode(Cur that, String text) {
     // change from a phantom ref to a soft/weak ref so I can know what
     // to do when I dequeue from the old q.
 
-    public void _dispose() {
```

Comment:
does this need to change name? if so, can you keep the old method too and deprecate it?

Reply:
I renamed it only for consistency. In don't know why the `_dispose()` method even have to be public since it is only invoked as a method reference inside the same Cursor class, at the new `close()` that is renamed from old `dispose()`

Reply:
I may add that probably all those underscore prefixed methods don't need to be public.

Reply:
the method is already public so it cannot be renamed - all you do is add a 2nd method and deprecate the old one

Reply:
the new one can be private if it doesn't need to be public

Reply:
Reverted the method rename. I thins all these undercore prefixed methods don't need to be public and since `Cursor` is a private implementation of `XmlCursor`, its public API should not be considered an user API of XMLBeans, maybe another JIRA for the future for this.